app/repeater/settings.py

```diff
@@ -131,3 +133,6 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+
+APPEND_SLASH=False
```

Comment:
If I understand correctly, this means there won't be automatic redirects from something like `/rest/repeater` to `/rest/repeater/`.
Why do we want to include it? (To be clear, I'm not arguing against it, just making sure I understand! :D )

Reply:
i had an issue with post request turning get
later resolved it with duplicate urls.
would remove this in my next pr attempt